anycloud/node/src/datastore.ts

```diff
@@ -34,31 +34,35 @@ export class DataStore {
     if (!dsKey) return undefined;
     if (this.isLocal) {
       if (dsKey in this.localDS) {
-        const val = this.localDS[dsKey]; 
-        if (typeof(val) === 'object') {
-          return JSON.stringify(val);
-        } else {
-          return `${val}`;
-        }
+        const val = this.localDS[dsKey];
+        return JSON.stringify(val);
```

Comment:
This should be `JSON.parse(val)`, right?

Reply:
Yes, I think you were reviewing an out of dated version, sorry.